.github/workflows/ci.yml

```diff
@@ -56,6 +56,7 @@ jobs:
             rustc: nightly
             allow_failure: true
             extra_args: --features=unstable
+          - os: ubuntu-22.04
```

Comment:
Note that `ubuntu-22.04` is not yet available on Github actions yet, but is [expected in early May](https://github.com/actions/virtual-environments/discussions/5427).

Reply:
Does this mean the PR can only merged then? I am unfortunately not so much involved in GitHub CI.

Reply:
It means that on GitHub the commit will not show the green checkmark next to it since not all tests have been executed. See https://github.com/Meinersbur/sccache/runs/6166559042. I don't know whether GitHub will retroactively make the `ubuntu-22.04` runners, once available, executed the pending jobs.